src/components/tables/Captures/Rows.tsx

```diff
@@ -63,25 +60,15 @@ function Row({ isSelected, setRow, row, stats, showEntityStatus }: RowProps) {
                 imageTag={`${row.connector_image_name}${row.connector_image_tag}`}
             />
 
-            {hasLength(tenantDetails) ? (
-                <>
-                    <Bytes
-                        val={
-                            stats
-                                ? stats[row.catalog_name]?.bytes_written_by_me
-                                : null
-                        }
-                    />
+            <Bytes
```

Comment:
This is an attempt to both handle issues where a browser translating the app could crash stuff. Also, makes the code a lot simpler to think through as now we will always have the same columns and just hide them when needed.